recipe/conda_build_config.yaml

```diff
@@ -0,0 +1,2 @@
+channel_sources:
+- freecad,conda-forge
```

Comment:
this is not ok at conda forge

Reply:
as indicated in the commit message that is temporary so I can already see if it builds while 5.15 was not yet in conda-forge

Reply:
removed, this has to wait for aarch64 and ppc64le qt-main packages before merge